add WAF steps to user documentation (#389)
moj-analytical-services/user-guidance: source/documentation/apps/rshiny-app.md

```

A full working example is available [here](https://github.com/ministryofjustice/ap-rshiny-notesbook)

#### Enabling Web Application Firewall (WAF) for Data Platform Apps 

There are two flags that need to be set to enable WAF for your application, Ingress.ModSec.enabled and GithubTeam

> To do this by adding the following lines in to your app's  `build-push-deploy-dev.yml` and `build-push-deploy-prod.yml` GitHub action workflow files:
> 
> To change the resources insert at  the end of the file just before the $custom_variables the following as appropriate, remembering to use the line
> continuation “\” on the existing last line.

Reviewer: ```suggestion
To do this modify your app's  `build-push-deploy-dev.yml` and `build-push-deploy-prod.yml` GitHub action workflow files as follows:
```

Reviewer: Made a small tweak to language.
